ui/raidboss/data/07-dt/raid/r5n.txt

```diff
@@ -1,9 +1,242 @@
-### Arcadion (R5N): AAC Cruiserweight M1
+### AAC CRUISERWEIGHT (M1)
+# ZoneId: 1256
+
+# @TODO: There are definitely more IDs for 2-snap and 4-snap twist, see:
+# https://v2.xivapi.com/api/search?sheets=Action&fields=Name,Cast100ms&query=Name=%222-snap%20Twist%22
+# https://v2.xivapi.com/api/search?sheets=Action&fields=Name,Cast100ms&query=Name=%224-snap%20Twist%22
```

Comment:
Are these specific to normal mode, or is it more probable that some of these are for Savage use?

Reply:
I had additional IDs in my log files but didn't have footage to confirm their patterns.